specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-02-01/managedClusters.json

```diff
@@ -2848,6 +2848,11 @@
           "type": "string",
           "description": "FQDN of private cluster."
         },
+        "azurePortalFQDN": {
+          "readOnly": true,
+          "type": "string",
+          "description": "FQDN for the master pool which used by proxy config."
```

Comment:
what do you mean by "master pool"?

Reply:
ccp... keep it consistent as description from fqdn.